lib/hope-tab.js

```diff
@@ -63,6 +66,43 @@ window.HopeTab = fxosComponent.register('hope-tab', {
     this.url = url;
   },
 
+  onScroll(e) {
```

Comment:
nit: onBrowserScroll

(I'm afraid we'll start to get confused at some point :))